### ISSUE TITLE
MiUI/Redmi Cleaner app causing Pano to not scrobble

### ISSUE THREAD

**Issue by FateXBlood** [kawaiiDango/pano-scrobbler#463]
**Describe the bug**
Redmi has an in-built Cleaner app that cleans all sorts of trash and cache from the device. During this process, it also seems to remove certain things from the Memory about Pano Scrobbler, which is crucial for song detection.

**Steps to Reproduce**
Make the app compatible so as not to let the Cleaner app detect Pano Scrobbler's activity and not to let it remove the data from the Memory.

**App version**
3.27 - 2024, Mar 25

**Comment by kawaiiDango** [comment]
There is nothing an app developer can do about this. See https://issuetracker.google.com/issues/122098785

**Comment by kawaiiDango** [comment]
Quote from the first comment in the Google issue tracker: 

> Additionally, these OEMs preload (and remotely configure) a whitelist of packages that are allowed to freely perform said actions. How is this fair to the rest of us?

**Comment by serious-angel** [comment]
> Quote from the first comment in the Google issue tracker:
> 
> > Additionally, these OEMs preload (and remotely configure) a whitelist of packages that are allowed to freely perform said actions. How is this fair to the rest of us?

Thank you, indeed I see, but why doesn't it happen in the version v3?

I do imagine that the changes are huge in comparison between the versions, considering the change-logs, and I do heartfelt appreciate you for it, for investing time, money, and effort in modernizing and enhancing your, also awesome lovely, project, absolutely. It is always important, of course, regardless.

**Comment by serious-angel** [comment]
> This issue was created on May 11, 2024 when v4 was not even released.

It is a different case then indeed. To support the case of MIUI, I tried the most recent v3 version `v3.45`, APK from the repository Releases, with the following device, and it had been working like a charm for several weeks tried (I specifically disabled Auto-Update in Play Market):
```
Model name: Xiaomi MI 10T Pro
Model:      M2007JS3G
OS:         MIUI Global 14.0.1 Stable
Android:    12 SKQ1.211006.001
```

I just did the common generals for the Pano Scrobbler in settings:
1. Locked in Current Tasks screen to not terminate manually by an accident.
2. Disabled Battery optimizations;
3. Enabled background autostart.

This v3 version never got unexpectedly terminated by MIUI, and always handles the scrobbles correctly from Poweramp.

The Play Market purchase activation worked for the side-loaded APK, too, and I appreciate you for it to have professionally done so it works flawlessly.
And, the JWT got at Ko-Fi is supported in both versions for PC, and I appreciate you for it heartfelt, too!
